docs/CHANGELOG.rst

```diff
@@ -28,6 +28,7 @@ Fixed
 -----
 * Enable DEBUG setting in tests using Django's LiveServerTestCase
 * Wait for ElasticSeach to index the data in KBBioProcessTestCase
+* Processes for unused parameters: Tophat, chip-seq workflow
```

Comment:
Maybe better: Add/remove unused parameters in processes `alignment-tophat2` and workflow `workflow-chip-seq`